iam.tf

```diff
@@ -82,18 +83,6 @@ data "aws_iam_policy_document" "this" {
     }
   }
 
-  statement {
```

Comment:
Why we remove this policy ? According to the [exmaple policy](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/example-iam-policy.json) it is recommended

![image](https://github.com/lablabs/terraform-aws-eks-ebs-csi-driver/assets/6748956/b20c5f91-452d-4657-80ab-5d8c866c550f)